lib/factory_bot/definition_proxy.rb

```diff
@@ -221,6 +221,50 @@ def trait(name, &block)
     #     end
     #   end
     #
+    # Prefix and Suffixes can be used for trait definition.
+    # both can be set to true (prefix/suffix will be set to enum name), or to a custom string
+    #
+    # Example:
+    #   factory :task do
+    #     traits_for_enum :status, [:started, :finished], _prefix: true
+    #   end
+    #
+    # Equivalent to:
+    #   factory :task do
+    #     traits_for_enum :status, [:started, :finished], _prefix: 'status'
+    #   end
+    #
+    # Both Equivalent to:
+    #   factory :task do
+    #     trait :status_started do
+    #       status { :started }
+    #     end
+    #
+    #     trait :status_finished do
+    #       status { :finished }
+    #     end
+    #   end
+    #
+    # Example:
+    #   factory :task do
+    #     traits_for_enum :status, [:started, :finished], _suffix: true
+    #   end
+    #
+    # Equivalent to:
+    #   factory :task do
+    #     traits_for_enum :status, [:started, :finished], _suffix: 'status'
```

Comment:
Might be worth pointing out we would need to update the docs here if we end up using the signature without the hash arguments.